server/templates/base.html

```diff
@@ -3,6 +3,7 @@
 <html lang="en">
 <head>
   {% block style %}{% endblock %}
+  {% block targetblank %}{% endblock %}
```

Comment:
Do we need this block? Why not put the content in directly?

Reply:
My thinking was: we definitely want all links that are clicked _within a notebook_ to open a new tab (because eval state could be lost), but we might not want links that are included on server pages to always open a new tab. It was my understanding that this template is used on both server pages and on the notebook. Is that incorrect? Or alternatively, even if that is correct, do you think we should always force links to open in a new tab?

Reply:
That totally makes sense to me, but you can just put the needed directive inside notebook.html, no need for the block.

Reply:
hey will, i'm not sure how to put this directly in notebook.html while also wrapping it in the <head> tag. Would we need to replace the `{% block style %}` with a  `{% block head %}` across the board? or is there a way of injecting something into a tag from within a child template?

Reply:
talked to @wlach -- the solution is to rename the "style" block to "head" and to but the <base> tag in the head block within the notebook template.